src/App.tsx

```diff
@@ -1,23 +1,92 @@
 import "./App.css";
-
+import React, { useState } from 'react';
 import { taskList } from "./taskList";
 
+let taskListCopy = taskList;
+
+// Create a list with additional field to mark deleted task
+  let taskListExt: any[] = [];
```

Comment:
Never use `any`, make sure to define an interface for this. You can refer to this answer https://stackoverflow.com/questions/25469244/how-can-i-define-an-interface-for-an-array-of-objects